docs/jwst/skymatch/README.rst

```diff
@@ -65,6 +67,21 @@ two other methods have been introduced: ``'globalmin'`` and
   algorithm to find a baseline sky value common to all input images
   and the ``'match'`` algorithm to "equalize" sky values among images.
 
+In methods that find sky background levels in each image (``'localmin'``) or
+a single level for all images (``'globalmin'``), usually statistics is
+computed using sigma-clipping. If input images contain vast swaths of empty
+sky, then the sigma-clipping algorithm should be able to automatically
+exclude (clip) contributions from bright compact sources.
+In this case the measured "sky background" is the
+measured signal level from the "empty sky". On the other hand, the
+``'match'`` method compares *total* signal levels integrated over those regions
+in the images that correspond to common ("overlap") regions on the celestial
+sphere for both images being compared (comparison is pair-wise).
+Often, this method is used when there are no large
+"empty sky" regions in the images such as when a large nebula occupies most
+part of the view. This method cannot measure "true background" but
```

Comment:
Need to add "or" in the middle of "... large nebula occupies most **or** part of the view."